rackcity/api/serializers/asset_serializers.py

```diff
@@ -121,20 +127,70 @@ class BulkAssetSerializer(serializers.ModelSerializer):
     )
     # by default, calls get_<field> - in this case, get_rack
     rack = serializers.SerializerMethodField()
-    rack_position = serializers.IntegerField(source='rack_position')
+    power_port_connection_1 = serializers.SerializerMethodField()
+    power_port_connection_2 = serializers.SerializerMethodField()
 
     class Meta:
         model = Asset
         fields = (
             'asset_number',
             'hostname',
+            'datacenter',
             'rack',
             'rack_position',
             'vendor',
             'model_number',
             'owner',
-            'comment'
+            'comment',
+            'power_port_connection_1',
+            'power_port_connection_2'
         )
 
     def get_rack(self, asset):
         return asset.rack.row_letter + str(asset.rack.rack_num)
+
+    def get_power_port_connection_1(self, asset):
+        return self.power_port_connection(asset, port_number=1)
+
+    def get_power_port_connection_2(self, asset):
```

Comment:
Do we want to generalize this somehow? What if there are more than 2 power ports

Reply:
The bulk format spec only has two columns for this, any additional connections would have to be specified manually